scalding/pom.xml

```diff
@@ -47,6 +47,8 @@
       <name>Concurrent Maven Repo</name>
       <url>https://conjars.org/repo</url>
     </repository>
+
+    <!-- the twitter repo is unreliable (https://github.com/twitter/hadoop-lzo/issues/148) -->
```

Comment:
In my understanding, Scalding is no longer under development. So I think we'd better remove this module itself. WDYT?

Reply:
@jongyoul jinjava is also used in submarine module - would it be possible to treat the possible removal of the scalding module as a separate issue?

Reply:
Of course, I just would like to say you don't have to consider the scalding module carefully. :-) I'll make a new PR to remove Scalding module itself.